manifests/vrrp/instance.pp

```diff
@@ -206,6 +211,7 @@
   $use_vmac                                                               = false,
   $vmac_xmit_base                                                         = true,
   Boolean $native_ipv6                                                    = false,
+
```

Comment:
was there a reason for this change? I think a puppet-lint plugin isn't happy about the new line here